Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycScreen.swift

```diff
@@ -53,7 +54,7 @@ struct OrchestratedBiometricKycScreen: View {
                 extraPartnerParams: extraPartnerParams,
                 skipApiSubmission: true,
                 onResult: viewModel
-            )
+            ).environmentObject(localMetadata)
```

Comment:
I see this injected in several places, but I couldn't see there the injected object was used in the views.